hyperjumptech/monika-config-generator
# feat(yaml): add yaml config file generator

# Monika Config Generator PR
Fix #89 add yaml config file generator

## What this PR is trying to solve
Change the json config file into yaml config file generation, since monika is now able to read from yaml file.

## How this PR solve the issue
1. set the file generated into yaml 
2. read from existing config file in yaml format

## How to test
1. run the application `npm run build & npm run start` 
2. open `http://localhost:3000`
3. follow the instruction until download config file
4. downloaded file should be in monika.yml
